msteams-platform/get-started/prerequisites.md

```diff
@@ -0,0 +1,204 @@
+- If you' a're developing with .NET, install [Visual Studio 2019](https://visualstudio.com/download).  Ensure you install the **ASP.NET and web development** or **.NET Core cross-platform development** workload.
+
+> [!WARNING]
+> There are known issues with `npm@7`, packaged with Node v15 and later. If you have problems running `npm install`, ensure you're using Node v14 (LTS)
+
+## Install the Teams Toolkit
+
+The Teams Toolkit helps simplify the development process with tools to provision and deploy cloud resources for your app, publish to the Teams store, and more. You can use the toolkit with Visual Studio Code, Visual Studio, or as a CLI (called `teamsfx`).
+
+# [Visual Studio Code](#tab/vscode)
+
+1. Open Visual Studio Code.
+1. Select the Extensions view (**Ctrl+Shift+X** / **⌘⇧-X** or **View > Extensions**).
+1. In the search box, enter _Teams Toolkit_.
+1. Select on the green install button next to the Teams Toolkit.
+
+You also can find the Teams Toolkit on the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension).
+
+The following tools will be installed by the Visual Studio Code extension when they are needed.  If already installed, the installed version will be used instead.  If using Linux, you must install these tools before use:
+
+- [Azure Functions Core Tools](/azure/azure-functions/functions-run-local)
+
+    Azure Functions Core Tools is used to run any backend components locally during a local debug run, including the authentication helpers required when running your services in Azure.  It is installed within the project directory (using the npm `devDependencies`).
+
+- [.NET SDK](/dotnet/core/install/)
+
+    The .NET SDK is used to install customized bindings for local debugging and Azure Functions app deployments.  If you have not installed the .NET 3.1 (or later) SDK globally, the portable version will be installed.
+
+- [ngrok](https://ngrok.com/download)
+
+    Some Teams app features (conversational bots, messaging extensions, and incoming webhooks) require inbound connections.  You need to expose your development system to Teams through a tunnel.  A tunnel is not required for apps that only include tabs.  This package is installed within the project directory (using npm `devDependencies`).
+
+# [Visual Studio 2019](#tab/vs)
+
+You can use Visual Studio 2019 to develop Teams apps with Blazor Server in .NET.  If you're not intending to develop Teams apps in .NET, install the Visual Studio Code version of Teams Toolkit.
+
+To install the Teams Toolkit extension:
+
+1. Open Visual Studio 2019.
+1. Select **Extensions** > **Manage Extensions**.
+1. In the search box, enter _Teams Toolkit_.
+1. Select the Teams Toolkit extension and select **Download**.
+
+The extension will be downloaded.  Close Visual Studio 2019 to install the extension.
+
+# [Command line](#tab/cli)
```

Comment:
In fact, `teamsfx-cli` also needs the same dependencies (.NET SDK, ngork, func tool) as `Visual Studio Code (Teams Toolkit)`